### PR TITLE
doc: break into 2 sentences

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/libstd/thread/mod.rs [hunk 1]
```diff
@@ -245,9 +245,9 @@ impl Builder {
     /// configuration methods can be chained.
     ///
     /// If the [`stack_size`] field is not specified, the stack size
-    /// will be the `RUST_MIN_STACK` environment variable, if it is
-    /// not specified either, a sensible default size will be set (2MB as
-    /// of the writting of this doc).
+    /// will be the `RUST_MIN_STACK` environment variable.
+    /// If it is not specified either, a sensible default will be set
```
Comment (steveklabnik) [inline src/libstd/thread/mod.rs]: could you re-flow this? I agree splitting the sentences makes a lot of sense, but now it's got weird newlines.

**Review by steveklabnik — CHANGES_REQUESTED** [review]
r=me with this reflowed properly

**Comment by tshepang** [comment]
is done

**Comment by steveklabnik** [comment]
@bors: r+ rollup

thank you ❤️

**Pull request merged**